src/test/K6/src/tests/app/performance/efm.js

```diff
@@ -0,0 +1,115 @@
+/*
+  Create and archive instances of RF-0002 without attachments
+  Test data: a json file named as ex: users_prod.json with user data in below format in the K6/src/data folder and deployed RF-0002 app
+  [
+    {
+        "username": "",
+        "password": "",
+        "partyid": ""
+    }
+  ]
+  example: k6 run -i 20 --duration 1m --logformat raw --console-output=./src/data/instances.csv src/tests/app/efm.js
+  -e appsaccesskey=*** -e archive=true -e delete=false -e harddelete=true
+*/
+
+import { check } from 'k6';
+import { SharedArray } from 'k6/data';
+import { addErrorCount, stopIterationOnFail } from '../../../errorcounter.js';
+import * as appInstances from '../../../api/app/instances.js';
+import * as appData from '../../../api/app/data.js';
+import * as appProcess from '../../../api/app/process.js';
+import * as platformInstances from '../../../api/platform/storage/instances.js';
+import { deleteSblInstance } from '../../../api/platform/storage/messageboxinstances.js';
+import * as setUpData from '../../../setup.js';
+
+const appOwner = "ttd";
+const level2App = "eformidling-app";
+const environment = "tt02";
```

Comment:
this has to be changed to `const environment = __ENV.env.toLowerCase();`

Reply:
Hard coded as we don't have aother environments to support at the moment. Not sure why we would require these tests in produciton? So I'd rather only open up for testing i TT as we don't want to disrupt any of the systems on the receiving end in production